app/queues/dns/workers/dns-update-worker.server.ts

```diff
@@ -0,0 +1,49 @@
+import { UnrecoverableError, Worker } from 'bullmq';
+
+import { redis } from '~/lib/redis.server';
+import logger from '~/lib/logger.server';
+import { createRecord, upsertRecord, deleteRecord } from '~/lib/dns.server';
+
+import type { WorkType } from '../add-record-flow.server';
+import type { Record } from '@prisma/client';
+
+export const dnsUpdateQueueName = 'dns-record-update';
+
+export interface DnsUpdaterData {
+  workType: WorkType;
+  username: Record['username'];
+  type: Record['type'];
+  fqdn: string;
+  value: Record['value'];
+}
+
+export type dnsRecordUpdateJobResult = string;
```

Comment:
I think this should start with a `D` vs. `d`

Reply:
Will be fixed